Twitch/Twitch.Base/Models/Clients/Chat/ChatWhisperMessagePacketModel.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Collections.Generic;
+
+namespace Twitch.Base.Models.Clients.Chat
+{
+    /// <summary>
+    /// Information about a Chat whisper message packet.
+    /// </summary>
+    public class ChatWhisperMessagePacketModel : ChatPacketModelBase
```

Comment:
All of the information looks identical to what can be found in ChatMessagePacketModel, so just have this class inherent from that and new the CommandID.

Reply:
Will do